[Docs][Security] Explain how to limit repeated requests to public APIs (Rate Limit middleware)

squareloop1 opened the pull request:
# Issue
Resolves #384 

# Solution and steps

Adds explanation on how to use a rate limiter to the docs cookbook section.

# Checklist
- [x] Add/update/check docs (code comments and docs/ folder).

LoicPoullain commented:
Perfect!

Maybe we can also mention that the file to update is `src/index.ts`. Something like this?

*src/index.ts*
```typescript
import * as express from 'express';
import * as rateLimit from 'express-rate-limit';

...

async function main() {
  ...

  const expressApp = express();
  expressApp.use(rateLimit({
    windowMs: 15 * 60 * 1000, // 15 minutes
    max: 100 // limit each IP to 100 requests per windowMs
  }));

  const app = createApp(AppController, expressApp); // v1
  // For v0.8
  // const app = createApp(AppController, { /* ... */ }, expressApp);

  ...
}

main();
```

squareloop1 commented:
Good idea, didn't think of that.
I added a commit and extended the code to make it more obvious on where the rate limiter needs to be initialized.

Also changed title and file name for to be more consistent.
